onnxruntime/core/platform/telemetry.cc

```diff
@@ -37,7 +37,7 @@ void Telemetry::LogSessionCreation(uint32_t session_id, int64_t ir_version, cons
                                    const std::string& model_graph_name,
                                    const std::unordered_map<std::string, std::string>& model_metadata,
                                    const std::string& loadedFrom, const std::vector<std::string>& execution_provider_ids,
-                                   bool use_fp16) const {
+                                   bool use_fp16, uint32_t projection) const {
```

Comment:
looks like you have to add a "ORT_UNUSED_PARAMETER(projection)" line below.

Reply:
Thanks for catching this.